src/Common/src/Interop/BSD/libc/Interop.readdir.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
There is no "BSD" interop until now. We discussed it in the FreeBSD team and feel that is not a good idea.

This file should either in the Unix folder, if it works for most Unix like platforms or in the FreeBSD one is if just works on (Free)BSD.